test/message-service.test.js

```diff
@@ -0,0 +1,47 @@
+describe('Test message service', () => {
+  let messageService
+  let rheaPromiseMock
+
+  beforeEach(async () => {
+    messageService = require('../server/services/message-service')
+    rheaPromiseMock = require('rhea-promise')
+    jest.mock('rhea-promise')

```

Comment:
Create the mock once at the top of the test suite (inside the `describe()` callback, outside `beforeEach()`). Reset the mock in `beforeEach()`. Then there's no need to `unmock` in `afterEach()`.